api/schema/resolvers/ContributorResolver.js

```diff
@@ -98,7 +98,7 @@ module.exports = {
             })
             return contributorOrganizations
         },
-        getGithubRepos: async (root, { organizationName, githubPageNumber }, { cookies, models }) => {
+        getGithubRepos: async (root, { githubPageNumber, accountId }, { cookies, models }) => {
```

Comment:
Instead of using `accountId` it is possible to use the `contributor_id`? (the pk value of the Contributors table)

Reply:
Right now the logic is using the accountId in order to check if the user selected an organization or an user from the list, if we use the contributor id we would need to check twice and the accountId would still be necessary to see if its an org or an user